File: rebus-anagraficasrv/src/java/it/csi/rebus/anagraficasrv/vo/AmbTipServizioVO.java
/*
* SPDX-FileCopyrightText: (C) Copyright 2022 Regione Piemonte
* SPDX-License-Identifier: EUPL-1.2
*/
package it.csi.rebus.anagraficasrv.vo;

@SuppressWarnings("serial")
public class AmbTipServizioVO extends ParentVO {
	private Long idAmbTipServizio;
	private AmbitoServizioVO ambitoServizio;
	private TipologiaServizioVO tipologiaServizio;

	public Long getIdAmbTipServizio() {
		return idAmbTipServizio;
	}

	public void setIdAmbTipServizio(Long idAmbTipServizio) {
		this.idAmbTipServizio = idAmbTipServizio;
	}

	public AmbitoServizioVO getAmbitoServizio() {
		return ambitoServizio;
	}

	public void setAmbitoServizio(AmbitoServizioVO ambitoServizio) {
		this.ambitoServizio = ambitoServizio;
	}

	public TipologiaServizioVO getTipologiaServizio() {
		return tipologiaServizio;
	}

	public void setTipologiaServizio(TipologiaServizioVO tipologiaServizio) {
		this.tipologiaServizio = tipologiaServizio;
	}

	
}

File: rebus-anagraficawcl/src/app/vo/amb-tip-servizio-vo.ts
/*************************************************
Copyright Regione Piemonte - 2022
SPDX-License-Identifier: EUPL-1.2-or-later
***************************************************/
import { AmbitoServizioVO, TipologiaServizioVO } from './drop-down-menu-vo';

export class AmbTipServizioVO {

    public idAmbTipServizio: number;
    public ambitoServizio: AmbitoServizioVO;
    public tipologiaServizio: TipologiaServizioVO;
    public selected: boolean;

    constructor(idAmbTipServizio?: number, ambitoServizio?: AmbitoServizioVO, tipologiaServizio?: TipologiaServizioVO, selected?: boolean) {
        this.idAmbTipServizio = idAmbTipServizio;
        this.ambitoServizio = ambitoServizio;
        this.tipologiaServizio = tipologiaServizio;
        this.selected = selected;
    }
}